examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala

```diff
@@ -21,15 +21,12 @@ import java.text.BreakIterator
 
 import scala.collection.mutable
 
-import scopt.OptionParser
-
 import org.apache.log4j.{Level, Logger}
-
-import org.apache.spark.{SparkContext, SparkConf}
 import org.apache.spark.mllib.clustering.LDA
 import org.apache.spark.mllib.linalg.{Vector, Vectors}
 import org.apache.spark.rdd.RDD
-
+import org.apache.spark.{SparkConf, SparkContext}
```

Comment:
The Spark imports were correctly ordered already. I think the scopt import could go after the log4j import, if we're matching style of other imports.